README.md

```diff
@@ -11,20 +11,19 @@ These core user-facing software include:
 - [JupyterHub](https://github.com/jupyterhub/jupyterhub)
 
 > **Important notice**
-> As of March 2019 this software **is not accessible.**
-> Significant work will be required to reach that goal.
+> As of March 2022 this software **is not accessible.**
+> Significant work will be required to reach that goal. In the meantime, you can track the progress and initiatives in our [Jupyter Accessibility Team Compass](https://jupyter-accessibility.readthedocs.io)
```

Comment:
minor non-blocking suggestion - I've found it easiest to use [semantic line breaks](https://sembr.org/) to make diffs for docs PRs easier to manage and read. My rule is generally "one line per sentence".

Reply:
Agree - 👍🏽  will fix this once done with the rest of the bulk